assignment-1/pkg/apis/shahpratikr.dev/v1alpha1/types.go

```diff
@@ -0,0 +1,32 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type Snapshot struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              SnapshotSpec   `json:"spec,omitempty"`
+	Status            SnapshotStatus `json:"status,omitempty"`
+}
+
+type SnapshotSpec struct {
+	PvcName   string `json:"pvcname,omitempty"`
```

Comment:
```suggestion
	PVCName   string `json:"pvcname,omitempty"`
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms